fisher.fish

```diff
@@ -117,7 +117,7 @@ function $fisher_cmd_name -d "fish plugin manager"
     command mkdir -p {"$fish_path","$fish_config"}/{conf.d,functions,completions} "$fisher_config" "$fisher_cache"
     or return 1
 
-    set -l completions "$fish_config/completions/$fisher_cmd_name.fish"
+    set -l completions "$fish_path/completions/$fisher_cmd_name.fish"
```

Comment:
@externl It isn't, but you are responsible for setting up fish_path and adding dirs to your $fish_{function/complete}_path yourself as well as loading snippets on startup (which I see you are doing).

Hmm 🤔 🙃

I think this might be solid.